tests/test_backend_patent_uspto.py

```diff
@@ -110,12 +111,11 @@ def test_patent_groundtruth(patents, groundtruth):
             assert (
                 pred_md == gt_names[md_name]
             ), f"Markdown file mismatch against groundtruth {md_name}"
-        json_name = path.stem + ".json"
-        if json_name in gt_names:
-            pred_json = json.dumps(doc.export_to_dict(), indent=2)
-            assert (
-                pred_json == gt_names[json_name]
-            ), f"JSON file mismatch against groundtruth {json_name}"
+        json_path = path.with_suffix(".json")
+        if json_path.stem in gt_names:
+            assert verify_document(
+                doc, str(json_path), False
```

Comment:
is this on purpose or should be changed to `GENERATE`?

```suggestion
                doc, str(json_path), GENERATE
```